crates/goose-cli/src/log_usage.rs

```diff
@@ -0,0 +1,83 @@
+use goose::providers::base::Usage;
+
+#[derive(Debug, serde::Serialize, serde::Deserialize)]
+struct SessionLog {
+    session_file: String,
+    usage: goose::providers::base::Usage,
+}
+
+pub fn log_usage(session_file: String, usage: Usage) {
+    let log = SessionLog {
+        session_file,
+        usage,
+    };
+
+    // Ensure log directory exists
+    if let Some(home_dir) = dirs::home_dir() {
+        let log_dir = home_dir.join(".config").join("goose").join("logs");
+        if let Err(e) = std::fs::create_dir_all(&log_dir) {
+            eprintln!("Failed to create log directory: {}", e);
+            return;
+        }
+
+        let log_file = log_dir.join("goose.log");
+        let serialized = match serde_json::to_string(&log) {
```

Comment:
the old format had a data which was nice for a quick look/check without trying to match up the session name

Reply:
oh wait, looks like i have different versions. see this too 

2024-12-09 17:14:50,610 - goose - INFO - Session name: w2h9 | Cost for model claude-3-5-sonnet-2 Usage(input_tokens=17274, output_tokens=567, total_tokens=17841): $0.06